docs/compression_algorithms/Sparsity.md

```diff
@@ -81,13 +81,13 @@ The magnitude sparsity method implements a naive approach that is based on the a
             "num_bn_forget_steps": 5, // Number of batches from the training dataset to pass through the model at initialization in order to erase batchnorm statistics of the original model (using large momentum value for rolling mean updates)
         }
     }
+    "sparsity_init": 0.05,// "Initial value of the sparsity level applied to the model in 'create_compressed_model' function
     "params": {
             "schedule": "multistep",  // The type of scheduling to use for adjusting the target sparsity level
             "patience": 3, // A regular patience parameter for the scheduler, as for any other standard scheduler. Specified in units of scheduler steps.
-            "sparsity_init": 0.05,// "Initial value of the sparsity level applied to the model
             "sparsity_target": 0.7, // Target value of the sparsity level for the model
-            "sparsity_target_epoch": 3, // The target sparsity value will be reached after this many epoch steps
-            "sparsity_freeze_epoch": 50, // The number of steps after which the sparsity mask will be frozen and no longer trained
+            "sparsity_target_epoch": 3, // Index of the epoch from which the sparsity level of the model will be equal to spatsity_target value
```

Comment:
1. I would like to clarify how should it work in case of per-step sparsity scheduler: will sparsity rich target value at the first step of `sparsity_target_epoch` or at the last step, @vshampor what do you think?

Reply:
Should probably later make `*_target_step` and `*_target_epoch` settings mutually exclusive. The step is calculated globally, across all epochs. I'm not sure that we even have a `*_target_step` option.

Reply:
I'm not sure whether we need `*_target_step` or not it looks like new feature, but for `*_target_epoch` (for per-step case) I would like to clarify which training step of `*_target_epoch` epoch will be trained with target sparsity value the first or the last. It seems to me that it should be the last step of `*_target_epoch`. If we agree on that lets state it?